server/src/index.mjs

```diff
@@ -59,6 +60,19 @@ server.on('upgrade', function upgrade(request, socket, head) {
   }
 });
 
+
+//Fetch the yaml files from MFOS_Call_Flow repo
+const yamlPath = ".././cli/examples";
+shell.cd(yamlPath)
+try{
+  shell.exec('git clone git+ssh://git@github.comcast.com/lightning-automation/MFOS_Call_Flow')
```

Comment:
We don't want this within Mock Firebolt... it would be a dependency from public github to Comcast's GHE.

Reply:
removed the code

Reply:
removed the code